modules/collectd/manifests/plugin/rabbitmq.pp

```diff
@@ -1,31 +1,23 @@
 # == Class: collectd::plugin::rabbitmq
 #
-# Sets up a collectd plugin to monitor RabbitMQ.
+# Sets up the NYTimes collectd-rabbitmq collectd plugin to monitor RabbitMQ.
 #
 # === Parameters
 #
 # [*monitoring_password*]
-#   The password to a user which can access monitoring information from RabbitMQ.
+#   The password to the 'monitoring' user which can access monitoring information from RabbitMQ.
 #
 class collectd::plugin::rabbitmq (
   $monitoring_password,
   ){
-  include collectd::plugin::python
 
-  @file { '/usr/lib/collectd/python/rabbitmq.py':
-    ensure => present,
-    source => 'puppet:///modules/collectd/usr/lib/collectd/python/rabbitmq.py',
-    tag    => 'collectd::plugin',
-    notify => File['/etc/collectd/conf.d/rabbitmq.conf'],
-  }
-
-  @file { '/usr/lib/collectd/python/rabbitmq.pyc':
-    ensure => undef,
-    tag    => 'collectd::plugin',
+  package { 'collectd-rabbitmq':
+    ensure   => present,
```

Comment:
Could you specify a version of the package here (eg `ensure => '1.2.3'`). This prevents the case where we build a new machine which installs the latest version of the package rather than a version we know is good.